### ISSUE TITLE
Support offline mobile

### ISSUE THREAD

**Issue by vnugent** [OpenBeta/open-tacos#454]
Umbrella issue for improving mobile and offline support.
- [ ] https://github.com/OpenBeta/open-tacos/issues/493

**Comment by CocoisBuggy** [comment]
I'm gonna start by working on some simple problems

1. Get to the point where code can be saved locally on devices. Chrome lets you dowload as PWAs and android lets you "Save to homescreen". This won't do anything except show offline errors
2. Provision some systems for interacting with local storage in a slightly easier way
3. Work out client storage of maps
4. Add UI buttons for saving areas / climbs offline
5. Potentially look at saving local changes for later review...

A fair number of these are not blocked by any other outstanding features, it's mostly just some client side routing concerns that need to be worried about and worked out

**Comment by vnugent** [comment]
### WIP TODO list:

Sort term:

- [x] Add icon sizes
- [ ] Prevent unwanted caching (data, images)
- [ ] Mark custom offline cache (and actually fetch them)
  - Size limit?
- [ ] Solve annoying error on custom url's
- [ ] Discard cache on upgrade
- [ ] Detect offline
- [ ] Simplified homescreen on offline
- [ ] Disable all editing features when offline

Long term:

- [ ] Sirv custom caching 
- [ ] Modifications
- [ ] Image "utility" annotation
- [ ] Map support
